app/src/main/res/layout/dashboard_spotlight_concat.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
change the name of this layout to be `dashboard_tools_spotlight`, the `concat` is unnecessary, and just `dashboard_spotlight` doesn't indicate that it's part of the tools UI